### PR TITLE
Upload transations automatically only if master

### PR DESCRIPTION
**Pull request by McGiverGim**
This PR changes the upload of messages file to Crowdin, only when the merge is to the master branch.

Without this, if we merge to some maintenance branch, it would upload the file to a maintenance branch in Crowdin. This is correct, but can confuse users. If we need to do this at some moment, we can use the manual launch and will work.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
